hq/public/stylesheets/main.css

```diff
@@ -173,12 +177,31 @@ nav {
 .sg-details__header {
     background-color: #009688;
     color: #ffffff;
+    display: flex;
+    line-height: 36px;
+    min-height: 36px;
+}
+
+.sg-details__header i {
+    display: inline-grid;
+    float: none;
+    height: 36px;
+    line-height: 36px;
+    margin-right: 0;
+    vertical-align: middle;
+}
+
+.sg--details__count {
+    margin-right: 5px;
+    text-align: end;
+    width: 30px;
 }
 
 .usage-cta {
     display: block;
     background-color: #4E8098;
     padding: 0 .6rem;
+    white-space: nowrap;
```

Comment:
This prevents long resource names from breaking onto a newline if they contain a hyphen.